web/modules/custom/vote_utility/translations/nv.po

```diff
@@ -13,214 +8,285 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\\n"
 "Plural-Forms: nplurals=2; plural=(n > 1);\\n"
 
-#language selector
+# config.languages.languageName
 msgid "Navajo"
 msgstr "Diné"
 
+# config.languages.params.select_language
 msgid "Select language"
 msgstr "Saad choid77['88[7g77"
 
-#main
+# config.languages.params.Owner
 msgid "@sitename in language"
-msgstr "@sitename Diné"
+msgstr "@sitename in Diné"
 
+# config.languages.params.go_back
 msgid "Go back"
 msgstr "N1t'33'"
 
-msgid "Illustration of voter box with checked ballot being inserted to indicate voting action"
-msgstr "&'ii'n77[ b1 sits'aa y1zh7 si'1n7g77 biyi' asdzoh naashch'22'7g77 47 n7l47 o'oot'ah n7igo yaa y1[ti'go si'32go 1lyaa"
-
+# config.languages.params.last_updated
 msgid "Last updated:"
-msgstr "Ak44'di n1zk'33z:"
+msgstr "Ak44'di n1zk'33z: "
+
+# config.languages.params.ext_link_title
+msgid "External link opens in new window"
+msgstr "External link opens new window"
 
-#identifier
+# config.languages.params.skip_text
+msgid "Skip to main content"
+msgstr "&yis77 baa hane'7gi nin7[keed"
+
+# config.languages.params.english_only
+msgid "(in English)"
+msgstr "&lt;Bilag1ana k'ehj7&gt;"
+
+# data.translations.footer.vote_logo_alt_text
+msgid "Illustration of voter box with checked ballot being inserted to indicate voting action"
+msgstr "&'ii'n77[ b1 sits'aa y1zh7 si'1n7g77 biyi' asdzoh naashch'22'7g77 47 n7l47 o'oot'ah n7igo yaa y1[ti'go si'32go 1lyaa "
+
+# data.translations.footer.identifier_gsa_txt
 msgid "An official website of the @link."
 msgstr "!l32j8' han1lki'7g77 website kod00 @link"
 
+# data.translations.footer.identifier_gsa_txt__1
 msgid "General Services Administration"
 msgstr "General Services Administration woly4ego t'11 a[tso yee 1k1 an7daalwo'd00 &lt;Bilag1ana k'ehj7&gt;ß"
```

Comment:
I looked at the original translation file and this character at the end needs to be removed `ß﻿`

Reply:
removed tailing character